content/en/cloud_cost_management/tags/_index.md

```diff
@@ -0,0 +1,126 @@
+| Cloud Cost Management | Cloud Cost Aliases | Some aliases are tags derived from provider cost data, used to simplify the cost data model, such as `aws_product`, which is an alias of `lineItem/ProductCode`. Additional tags that exist on both cost data and integration metrics are added, allowing cost data and usage data to be combined in Custom Allocation Rules, dashboards, and notebooks |
+| Cloud Cost Management | Cloud Cost Allocation | Tags created during cost allocation that specify the split of shared resources, such as `allocated_spend_type` |
+| Cloud Cost Management | FOCUS | Provider-agnostic tags compliant with [FOCUS][8], the unifying format for cloud billing data |
+| Tag Pipelines | Rules defined by user | Tags created by applying your Tag Pipelines to cost data |
+| Custom Allocation Rules | Rules defined by user | Tags created by applying your Custom Allocation Rules to cost data (do not apply to SaaS costs) |
+
+Datadog also adds provider specific tags:
+
+| Provider | What tags are collected | Description |
+|---|---|---|
+| AWS | Cost Allocation Tags | Any tag defined by the user in [AWS Cost Allocation][1] tags that show up in the AWS CUR |
+| AWS | AWS Resource Groups Tagging API | User-defined tags on a cloud resource in AWS, pulled by Cloud Cost Management using the Group Tagging API |
+| AWS | AWS Organizational Units | User-defined tags on an AWS organizational units using [AWS Organizations][7] |
+| AWS | AWS Organizations - Accounts | User-defined tags on an AWS account using [AWS Organizations][7] |
+| Amazon ECS | Amazon ECS Task | User-defined tags on an ECS task definition |
+| Amazon ECS | Amazon ECS Container | User-defined tags on a container running in an ECS task |
+| Azure | Azure Cost Export - User Resource Tags | User-defined tags on a cloud resource in Azure, found in the Tags column in the Azure cost export. Does not include Resource Group tags |
+| Azure | Azure Cost Export - System Resource Tags | Azure-defined tags on a cloud resource, found in the AdditionalInfo column in the Azure cost export |
+| Google Cloud | Google Billing Export - Project Labels| User-defined labels on a project in Google Cloud, found in the project.labels column in the billing export |
+| Google Cloud | Google Billing Export - System Resource Labels | System-generated labels on a resource in Google Cloud, found in the system_labels column in the billing export |
+| Google Cloud | Google Billing Export - User Resource Labels | User-defined labels on a cloud resource in Google Cloud, found in the labels column in the billing export |
+| Google Cloud | Google Billing Export - User Resource Tags | User-defined tags on a cloud resource in Google Cloud, found in the tags column in the billing export. The `goog-originating-sku-description` tag is added leveraging Google's SKU APis, to provide more granular SKU details for commitment line items. |
```

Comment:
`goog-originating-sku-description` is not really a user resource tag... technically isn't that one a billing dimension? but we don't have an obvious spot for it in these tables.